Bottom: e6c0a66d6dacbfd45680b5e065b7b1be88bb2d31
Top:    13525127cfd660ddc0cb1d9724872a4a8115ca20
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-19 19:31:42 +0000

Refresh of remove-debug-from-hairline

---

diff --git a/src/polygon1.c b/src/polygon1.c
index fa46ebd..d55ec3c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -416,37 +416,6 @@ node_add_single_point (VNODE * a, Vector p)
   return new_node;
 }				/* node_add_point */
 
-#if 1
-static void
-cvc_list_dump (CVCList *list)
-{
-  VNODE *node = list->parent;
-  CVCList *iter;
-  int count = 0;
-
-  pcb_fprintf (stderr, "Dumping CVC list at (%$mn, %$mn)\n", node->point[0], node->point[1]);
-
-  iter = list;
-  do {
-    count ++;
-    pcb_fprintf (stderr, "angle = %.30e, poly = %c, side = %c, (%mn, %mn)-(%mn, %mn), Vertices: %p-%p Edge: %p\n",
-                 iter->angle,
-                 iter->poly,
-                 iter->side,
-                 EDGE_BACKWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[0],
-                 EDGE_BACKWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[1],
-                 EDGE_FORWARD_VERTEX  (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[0],
-                 EDGE_FORWARD_VERTEX  (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[1],
-                 EDGE_BACKWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side)),
-                 EDGE_FORWARD_VERTEX  (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side)),
-                 VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side));
-  } while ((iter = iter->next) != list);
-
-  if ((count & 1) != 0)
-    g_error ("Ended up with odd number of entries in CVC list");
-}
-#endif
-
 /*
 edge_label
  (C) 2006 harry eaton
@@ -559,7 +528,6 @@ edge_label (VNODE * pn, int existing_label)
             g_critical ("Encountered identical edge in the other polygon (hairline edge pair)");
           else
             g_critical ("Encountered ambiguous edge in the other polygon (by compare_cvc_nodes), but geometry of each edge is different!");
-//          cvc_list_dump (first_l);
         }
 
       region = (l->side == 'P') ? INSIDE : OUTSIDE;
@@ -2532,7 +2500,6 @@ PLINE_check_hairline_edges (PLINE *contour)
               g_warning ("Wrapped around and found ourselves in the CVCList.. not quite sure how we managed that\n"
                          "Did we perhaps delete the start descriptor?");
 
-              cvc_list_dump (n);
               break;
             }
